packages/calcite-components/src/components/switch/switch.scss

```diff
@@ -1,3 +1,22 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-switch-background-color: Specifies the component's background color.
+ * @prop --calcite-switch-background-color-hover: Specifies the component's background color.
+ * @prop --calcite-switch-border-color: Specifies the component's border color.
+ * @prop --calcite-switch-corner-radius: Specifies the component's corner radius.
+ * @prop --calcite-switch-handle-background-color: Specifies the handle's background color.
+ * @prop --calcite-switch-handle-background-color-hover: Specifies the handle's background color.
```

Comment:
Because the :hover/:active is based on interaction with the parent .container not individual elements within the component this can be simplified to not include `-hover` tokens. You'll need to update the CSS so the pseudoclasses are on `:host`